### DISCUSSION TITLE
Question about environment

### DISCUSSION THREAD

**Discussion by zlj-cs** [PaddlePaddle/PaddleSpatial#45] [BaiduKDDCup2022] [answered]
As there is no classical tree model package in the released environments, can participants install other packages in conda environment, such as LightGBM or Xgboost?

**Comment by fersebas** [comment]
Can we use lightgbm and xgboost? please, clarify this...

**Reply by ramber1836** [reply]
Could you provide the detailed pip-requirements.txt or apt-requirements.txt as described in the README file?

**Comment by ramber1836** [comment, marked as answer]
LightGBM and XGBoost are supported by the system now.
